download/releaseNotes/releaseNotes7.adoc

```diff
@@ -6,6 +6,24 @@
 :awestruct-release_notes_version_qualifier: Final
 
 
+== New and noteworthy: Engine 7.20.0.Final
+
+=== Programmatic API made more convenient
+
+The programmatic API now supports a more fluent form too, because of the new `with*()` methods:
+
+[source,java,options="nowrap"]
+----
+solverFactory.getSolverConfig().withTerminationConfig(
+        new TerminationConfig().withMinutesSpentLimit(userInput));
+----
+
+=== Other
+
+* `@PlanningId` is now supported on primitive types such as `int` or `long` too, not just wrapper classes.
+* `Instant`, `Duration`, `Period` and other temporal classes can now be used as planning values in multithreaded solving.
```

Comment:
I would be careful with addressing thers as "other temporal classes" without further explanation. There is e.g. a package https://docs.oracle.com/javase/8/docs/api/java/time/temporal/package-frame.html with "temporal" in its name that we don't support (please correct me if I am wrong).

How about directly referring to the package we do support (java.time)?